### DISCUSSION TITLE
questions support how to run in yuzu

### DISCUSSION THREAD

**Discussion by dananothername** [xerpi/vita2hos#37] [Q&A] [answered]
https://github.com/xerpi/vita2hos/assets/101902085/58627907-a6cb-49e8-814c-6d168cb26b59

if this works in yuzu I'd make the first ps vita TAS

**Comment by DCNick3** [comment, marked as answer]
https://github.com/xerpi/vita2hos/blob/efb679892308ce8613f5396f2fd27293d3eb0d6c/README.md#running-it-on-yuzu

**Comment by dananothername** [comment]
let's continue in your discord
i already sent in #general
